sitl_run.sh fails trying to kill previous processes

It looks like the `sitl_run.sh` script fails to run `kill`. This is latest master (dc878f43a8e0841497abe7618e0142ac1f3baca2) on Arch Linux.

```
make posix jmavsim
...
SITL ARGS
sitl_bin: /home/julianoes/src/Firmware/build/posix_sitl_default/bin/px4
debugger: none
program: jmavsim
model: none
src_path: /home/julianoes/src/Firmware
build_path: /home/julianoes/src/Firmware/build/posix_sitl_default
empty model, setting iris as default
/home/julianoes/src/Firmware/Tools/sitl_run.sh: line 53: kill: juliano+: arguments must be process or job IDs
make[4]: *** [platforms/posix/CMakeFiles/jmavsim.dir/build.make:57: platforms/posix/CMakeFiles/jmavsim] Error 1
make[3]: *** [CMakeFiles/Makefile2:20122: platforms/posix/CMakeFiles/jmavsim.dir/all] Error 2
make[2]: *** [CMakeFiles/Makefile2:20129: platforms/posix/CMakeFiles/jmavsim.dir/rule] Error 2
make[1]: *** [Makefile:6281: jmavsim] Error 2
make: *** [Makefile:148: posix_sitl_default] Error 2
```

--- a/Tools/sitl_run.sh
+++ b/Tools/sitl_run.sh
@@ -47,7 +47,7 @@ pkill -x gazebo || true
 pkill -x px4 || true
 pkill -x px4_$model || true
 
-jmavsim_pid=`ps aux | grep java | grep Simulator | cut -d" " -f1`
+jmavsim_pid=`ps aux | grep java | grep "\-jar jmavsim_run.jar" | awk '{ print $2 }'`
 if [ -n "$jmavsim_pid" ]
 then
 	kill $jmavsim_pid

